quiche/src/lib.rs

```diff
@@ -8042,6 +8181,8 @@ pub struct TransportParams {
     pub retry_source_connection_id: Option<ConnectionId<'static>>,
     /// DATAGRAM frame extension parameter, if any.
     pub max_datagram_frame_size: Option<u64>,
+    /// Unknown peer transport parameters and values, if any.
+    pub unknown_params: Option<UnknownTransportParameters>,
```

Comment:
Let me know, @ghedo , if this was the type of change that you were suggesting. I _do_ think that it makes the code easier to read (and removes the need for that extra boolean). I sometimes don't like the ergonomics of working with `Option`s in Rust (when borrows are involved), but I think that your suggestion really made a big improvement. Again, please let me know if I captured your suggestion properly.